contents/dataVis.md

```diff
@@ -0,0 +1,390 @@
+            backgroundcolor = :white, xgridstyle=:dash, ygridstyle=:dash))
+    filename = "customPlot" # hide 
+    Options(current_figure(); filename, caption="custom plot", label="customplot") # hide
+    """
+sco(s)
+```
+
+This example has already most of the attributes that most users will play with it. 
+Probably a `legend` will also be good to have. Which for more than one function will 
+make more sense. So, let's  `append` another `plot object`, a mutating one `!`, and
+add the corresponding legends by calling `axislegend`. This will collect all the `labels`
+you might have passed to your plotting functions. 
+
+```jl
+s = """
+    lines(1:10, (1:10).^2, label = "x²", linewidth = 2, linestyle = nothing,
+        figure = (; figure_padding = 5, resolution = (600,400), 
+            backgroundcolor = :grey90, fontsize = 16, font = "sans"), 
+        axis = (; xlabel = "x", title = "title", backgroundcolor = :white, 
+            xgridstyle=:dash, ygridstyle=:dash))
+    scatterlines!(1:10, (10:-1:1).^2, label = "Reverse(x)²")
+    axislegend("legend", position = :ct)
+    current_figure()
+    filename = "customPlotLeg" # hide 
+    Options(current_figure(); filename, caption="custom plot legend", label="customplotlegend") # hide
+    """
+sco(s)
+```
+
+However, having to writte this so much code just for two lines can become cumbersome
+and tired, so if you plan on doing a lot of plots with the same general aesthetics
+then setting a theme will be better. We can do this with `set_theme!()` as the following
+example illustrates, not a particular good set of attributes but you get the idea. 
+
+```jl
+s = """
+    set_theme!(resolution = (600,400), backgroundcolor = (:orange, 0.5), 
+        fontsize = 16, font = "sans",
+        Axis = (backgroundcolor = :white, xgridstyle=:dash, ygridstyle=:dash),
+        Legend = (bgcolor = (:red,0.2), framecolor = :dodgerblue)
+        )
+
+    lines(1:10, (1:10).^2, label = "x²", linewidth = 2, linestyle = nothing,
+        axis = (; xlabel = "x", title = "title"))
+    scatterlines!(1:10, (10:-1:1).^2, label = "Reverse(x)²")
+    axislegend("legend", position = :ct)
+    current_figure()
+    set_theme!() # in order to go back to the default settings. 
+    filename = "setTheme" # hide 
+    Options(current_figure(); filename, caption="set theme", label="setTheme") # hide
+    """
+sco(s)
+```
+
+For more on `themes` please go to section xxx (Refs)
+
+Before moving on into the next section, it's worthwhile to see an example 
+where an `array` of attributes are passed at once to a plotting function. 
+For this example we will use the `scatter` plotting function to do a bubble plot. 
+
+```jl
+s = """
+    let
+        Random.seed!(123)
+        n = 100
+        fig, ax, pltobj = CairoMakie.scatter(randn(n), randn(n), color = randn(n), 
+            label = "Bubbles",colormap = :plasma, markersize = 25*rand(n), 
+            figure = (; resolution = (550,400)), axis = (; aspect= DataAspect()))
```

Comment:
```suggestion
            figure = (; resolution = (550,400)), axis = (; aspect = DataAspect()))
```
The named tuples do not adhere to the code style blue (https://github.com/invenia/BlueStyle#namedtuples), **but we can fix that later** :+1:.